doc/release/release_0.16.rst

```diff
@@ -0,0 +1,386 @@
+- Speed up ``skimage.feature.peak_local_max`` (#3984)
+- Better error message when incorrect value for keyword argument `kind` in
+  ``skimage.color.label2rgb`` (#4055)
+- All functions from ``skimage.drawing`` now supports multi-channel 2D images (#4134)
+
+API Changes
+-----------
+- Deprecated subpackage ``skimage.novice`` has been removed.
+- Default value of ``multichannel`` parameters has been set to False in
+  ``skimage.transform.rescale``, ``skimage.transform.pyramid_reduce``,
+  ``skimage.transform.pyramid_laplacian``,
+  ``skimage.transform.pyramid_gaussian``, and
+  ``skimage.transform.pyramid_expand``. No guessing is performed for 3D arrays
+  anymore, so, please, make sure that the parameter is fixed to a proper value.
+- Deprecated argument ``visualise`` has been removed from
+  ``skimage.feature.hog``. Use ``visualize`` instead.¨
+- ``skimage.transform.seam_carve`` has been completely removed from the
+  library due to licensing restrictions.
+- Parameter ``as_grey`` has been removed from ``skimage.data.load`` and
+  ``skimage.io.imread``. Use ``as_gray`` instead.
+- Parameter ``min_size`` has been removed from
+  ``skimage.morphology.remove_small_holes``. Use ``area_threshold`` instead.
+- Deprecated ``correct_mesh_orientation`` in ``skimage.measure`` has been
+  removed.
+- ``skimage.measure._regionprops`` has been completely switched to using
+  row-column coordinates. Old x-y interface is not longer available.
+- Default value of ``behavior`` parameter has been set to ``ndimage`` in
+  ``skimage.filters.median``.
+- Parameter ``flatten`` in `skimage.io.imread` has been removed in
+  favor of ``as_gray``.
+- Parameters ``Hxx, Hxy, Hyy`` have been removed from
+  ``skimage.feature.corner.hessian_matrix_eigvals`` in favor of ``H_elems``.
+- Default value of ``order`` parameter has been set to ``rc`` in
+  ``skimage.feature.hessian_matrix``.
+- ``skimage.util.img_as_*`` functions no longer raise precision and/or loss warnings.
+
+Bugfixes
+--------
+
+- Corrected error with scales attribute in ORB.detect_and_extract (#2835)
+  The scales attribute wasn't taking into account the mask, and thus was using
+  an incorrect array size.
+- Correct for bias in Inverse Radon Transform (``skimage.transform.irandon``) (#3067)
+  Fixed by using the Ramp filter equation in the spatial domain as described
+  in the reference
+- Fix a rounding issue that caused  a rotated image to have a
+  different size than the input (``skimage.transform.rotate``)  (#3173)
+- RANSAC uses random subsets of the original data and not bootstraps. (#3901,
+  #3915)
+- Canny now produces the same output regardless of dtype (#3919)
+- Geometry Transforms: avoid division by zero & some degenerate cases (#3926)
+- Fixed float32 support in denoise_bilateral and denoise_tv_bregman (#3936)
+- Fixed computation of Meijering filter and avoid ZeroDivisionError (#3957)
+- Fixed ``skimage.filters.threshold_li`` to prevent being stuck on stationnary
```

Comment:
```suggestion
- Fixed ``skimage.filters.threshold_li`` to prevent being stuck on stationary
```

Reply:
Good point.
As a side note, isn't there an easy way to check if you're stuck in a local maxima (cause that would be very bad…).